### DISCUSSION TITLE
Biometric compatible brands and models

### DISCUSSION THREAD

**Discussion by alejandrojourdan** [horilla-opensource/horilla#338] [General]
Please advise on biometric compatible brands and models
Thanks

**Comment by horilla-opensource** [comment]
> Please advise on biometric compatible brands and models Thanks

Hi @alejandrojourdan ,

Below is the list of biometric device brands and models that have been successfully tested and integrated with Horilla:
**ZKTeco Biometric Devices: **

- ZKTeco uFace
- ZKTeco K40 Pro
- ZKTeco SFace900 Semi-Outdoor
- ZKTeco FR1500 Finger & RFID Exit Reader (Flush Mounted)
- ZKTeco MB2000
- eSSL Biometric Devices:
- eSSL X2008
- eSSL X990

**Matrix COSEC Biometric Devices:**

- Matrix COSEC ARGO FACEE
- Matrix COSEC Path

**Anviz Biometric Devices:**   [Device supporting Anviz Software and Provide API]

**Reply by jain-divy** [reply]
How do I connect with eSSL X990? I only see the following options:
<img width="579" alt="image" src="https://github.com/user-attachments/assets/7bad5e61-afb8-4bee-9359-6caacad89e46">

**Reply by horilla-opensource** [reply]
Hi @jain-divy ,

You can select the ZKTeco Biometric for eSSL devices. Both use the same method.

With Regards,
**Team Horilla**

**Comment by alejandrojourdan** [comment]
Thank you very much!
